src/agent/onefuzz-agent/src/local/generic_analysis.rs

```diff
@@ -61,6 +62,7 @@ pub fn build_analysis_config(
 
 pub async fn run(args: &clap::ArgMatches<'_>) -> Result<()> {
     let common = build_common_config(args, true)?;
+    register_cleanup(common.job_id)?;
```

Comment:
can we make the behavior configurable with a paramter. I used 'keep_job_dir' in my local run ui PR